AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/bookings-list/bookings-list.component.spec.ts

```diff
@@ -1279,21 +1212,5 @@ describe('BookingsListComponent', () => {
             component.onChangeNoAllocated();
             expect(bookingPersistService.selectedUsers.length).toEqual(count);
         });
-
-        it('should not show allocated to label if work allocation feature flag is off', async () => {
-            launchDarklyServiceSpy.getFlag.withArgs(FeatureFlags.vhoWorkAllocation).and.returnValue(of(false));
-            await component.ngOnInit();
-            fixture.detectChanges();
-            const divToHide = fixture.debugElement.query(By.css('#allocated-to-' + bookingData.BookingsDetails[0].HearingId));
-            expect(divToHide).toBeFalsy();
-        });
-
-        it('should show allocated label to if work allocation feature flag is on', async () => {
```

Comment:
this test is still relevant, as work allocation is always on. We can remove the bit that sets the feature flag

Reply:
reinstated